pcdm-ext/use.rdf

```diff
@@ -40,8 +41,8 @@
     <rdfs:isDefinedBy rdf:resource="http://pcdm.org/use#"/>
   </rdfs:Class>
 
-  <rdfs:Class rdf:about="http://pcdm.org/use#PreservationMasterFile">
-    <rdfs:label xml:lang="en">preservation master file</rdfs:label>
+  <rdfs:Class rdf:about="http://pcdm.org/use#PreservationFile">
```

Comment:
Do we have a sense of the use of `pcdmuse:PreservationMasterFile`? Is there a need to indicate that the old class name is deprecated and replaced by `pcdmuse:PreservationFile`?